tiatoolbox/models/classification/patch_predictor.py

```diff
@@ -152,11 +152,9 @@ class CNNPatchPredictor:
         >>> data = [img1, img2]
         >>> predictor = CNNPatchPredictor(pretrained_model="resnet18-kather100k")
         >>> output = predictor.predict(data, mode='patch')
-
```

Comment:
keep the separating line to show the examples as separate.
You can also explain in comments what this example is doing.
e.g., 
`>>> # This example is an example for resenet-kather100k`
`>>> predictor = CNNPatchPredictor(input)`